3_visualize.yml

```diff
@@ -10,13 +10,16 @@ sources:
   - 3_visualize/src/svg_utils_general.R
```

Comment:
In order to build `public/date_peaks.csv` and `public/gw_sites.csv`, I had to add `data.table` and `stringr` to the packages for this yml file.

Reply:
added!